packages/devtools_app/pubspec.yaml

```diff
@@ -34,6 +34,7 @@ dependencies:
   http: ^1.1.0
   image: ^4.1.3
   intl: ^0.19.0
+  json_rpc_2: ^3.0.3
```

Comment:
why does this and stream_channel need to be moved up to dependencies instead of dev_dependencies. From what I can tell they are not used in production code

Reply:
Whoops, these were changes I had made while playing around with the widget preview client. I'll move them back.